### DISCUSSION TITLE
Work Around Plex limitation of only a single EPG provider

### DISCUSSION THREAD

**Discussion by mpking828** [chrisbenincasa/tunarr#1686] [General]
**Is your feature request related to a problem? Please describe.**
When adding tunarr to plex, the select XMLTV guide option is not available because a Plex Tuner is already selected.  Plex does not allow you to mix custom XLMTV and Plex EPG.

**Describe the solution you'd like**
- Ideal solution is that both the XMLTV for tunarr is used, as well as the Plex EPG, but that is up to Plex to fix. I recognize this is beyond tunarr control.
- tunarr to get an EPG from some (Schedules Direct?) source and combine them, and provide EPG.  This might be too big of an ask, since there will be so many corner cases, and lots of 3rd party software already doing this.
- Provide some guidance on using 3rd party stuff (or integrate some 3rd party stuff) to do this. Potentially might have to make the EPG exposed. _(I see this as the most likely solution)_
- Final case, document the issue in the install docs for the current state of the project.

**Describe alternatives you've considered**
I've added a bogus unused channel from my existing lineup, IE channel 1.   the Guide is useless, but the channel plays the content I created, I can just ignore that while "The Grinch" is playing, the EPG says "Cooking Christmas Cookies"

Currently investigating in not using Plex EPG, but using 3rd party software to download and EPG, and get the EPG from tunarr, and combine them:
https://www.reddit.com/r/PleX/comments/1fqhar7/both_custom_epg_and_cable_television_providers/

**Additional context**
Described here in reddit:
https://www.reddit.com/r/PleX/comments/17nudcq/add_xmltv_guide_option_missing/
https://www.reddit.com/r/PleX/comments/1fqhar7/both_custom_epg_and_cable_television_providers/

**Comment by Sonny-Crockett** [comment]
[UHF (TV iOS)](https://apps.apple.com/us/app/uhf-love-your-iptv/id6443751726), [TiviMate (Android TV)](https://tivimate.com/), [Open-TV (Windows Store)](https://github.com/Fredolx/open-tv), [StreamDock (Self Hosted Web-Based)](https://github.com/limmer55/streamdock)

Some client options for going around Plex, just in case. 

(edits for OS usage)

**Comment by chrisbenincasa** [comment]
I think this is out the scope of Tunarr, so I'm going to move it over to a discussion thread.
